.github/workflows/main.yml

```diff
@@ -0,0 +1,131 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  lint:
+    name: Check lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
```

Comment:
If we just specify `3`, what version is used ? I'd be inclined to do that if the version used is recent enough, so we don't need to remember to change this.

Reply:
> If we just specify 3, what version is used ?

The correct syntax is `3.x`, which according to the documentation uses the « latest minor release of Python 3 ». So currently it is the `3.8.5`. It will use `3.9` when `3.9` becomes stable if I understand correctly.

> I'd be inclined to do that if the version used is recent enough, so we don't need to remember to change this.

I agree, I will change that.